ext_emconf.php

```diff
@@ -17,12 +17,12 @@
 		'conflicts' => array(),
 		'suggests' => array(),
 	),
-	'state' => 'beta',
+	'state' => 'stable',
```

Comment:
State of a new version is in beta until a release is tagged

Reply:
On another note the version will be raised in a different commit so that change should probably be reverted